utils/scripts/terraform/openstack/requirements.sh

```diff
@@ -0,0 +1,7 @@
+sudo dnf install -y https://dl.google.com/linux/direct/google-chrome-stable_current_x86_64.rpm
+sudo dnf install -y python3-devel git redhat-lsb jq podman yum-utils java-1.8.0-openjdk
+sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo
```

Comment:
I think yum-config-manager is being replaced by `dnf config-manager`:
`sudo dnf config-manager --add-repo https://rpm.releases.hashicorp.com/RHEL/hashicorp.repo`

Reply:
incorporated the comments.